README.md

```diff
@@ -2,36 +2,43 @@
 
 ## Overview
 
-A brief description. Be clear about the purpose of the resolver,
-its capabilities and limitations.
+A custom Jinja filter for Sceptre for unquoting resolvers appearing in var files.
```

Comment:
You definitely should also include a description of the actual `unquote_resolvers` interface. It takes two additional parameters. What do those do? Why would you use those?

Reply:
Ok done